R/osmactive.R

```diff
@@ -28,7 +28,8 @@ et_active = function() {
     "tracktype",
     "surface",
     "smoothness",
-    "access"    
+    "access",
+    "route"
```

Comment:
Not sure about this: do you get any that are not NA? Seems route tags only apply to relations not segments: https://taginfo.openstreetmap.org/tags/route=bicycle#overview